src/FormElement/SelectOption.php

```diff
@@ -8,38 +8,61 @@ class SelectOption extends BaseHtmlElement
 {
     protected $tag = 'option';
 
-    /** @var mixed */
+    /** @var string|null Value of the option */
     protected $value;
 
+    /** @var string|null Label of the option */
+    protected $label;
+
     /**
      * SelectOption constructor.
      * @param string|null $value
      * @param string|null $label
      */
-    public function __construct($value = null, $label = null)
+    public function __construct(string $value = null, string $label = null)
```

Comment:
Then I'd recommend this: (and the consequences for the other parts that follow)

```suggestion
    public function __construct(?string $value, string $label)
```